java-processors/src/main/java/com/dataiku/dip/plugins/tradearea/PolygonBuilder.java

```diff
@@ -0,0 +1,26 @@
+package com.dataiku.dip.plugins.tradearea;
+
+import com.dataiku.dip.shaker.types.GeoPoint;
+
+public class PolygonBuilder {
```

Comment:
If there is no call to lineTo, the resulting polygon will be bogus (the 2 same points without a comma between the two).
Either document in the class comment that a lineTo call is mandatory, or handle the case.

Reply:
Done in 72e5797